src/query/storages/iceberg/src/database.rs

```diff
@@ -37,7 +37,7 @@ pub struct IcebergDatabase {
     ctl_name: String,
     /// operator pointing to the directory holding iceberg tables
     db_root: DataOperator,
-    /// database infomations
+    /// database information
```

Comment:
There's a typo checker in this repository, but it only complained about `informations` which seemed to be the default correction for `infomations` (which the typo checker didn't mind...). I'm happy to make this change as I agree the `-s` form isn't proper.